hackathon_site/event/serializers.py

```diff
@@ -127,6 +130,7 @@ def create(self, validated_data):
             "id_provided": False,
             "acknowledge_rules": acknowledge_rules,
             "e_signature": e_signature,
+            "phone_number": Application.objects.get(user=current_user).phone_number,
```

Comment:
in the `read_only_fields` of `CurrentProfileSerializer`, could you add phone_number to that list? because we don't want to change the phone number